locations/spiders/jigsaw_gb.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any
+
+from scrapy.http import Response
+from scrapy.spiders import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class JigsawGBSpider(Spider):
+    name = "jigsaw_gb"
+    item_attributes = {"brand": "Jigsaw", "brand_wikidata": "Q6192383"}
+    start_urls = ["https://jigsawimagestorage.blob.core.windows.net/jigsaw-logos/google-map-data-V3.json"]
+
+    def parse(self, response: Response, **kwargs: Any) -> Any:
+        for location in response.json()["features"]:
+            newlocation = {key.replace("branch_", ""): value for key, value in location["properties"].items()}
+            item = DictParser.parse(newlocation)
+            item["branch"] = newlocation["name"]
+            item["name"] = "Jigsaw"
+            item["street_address"] = newlocation["address_2"]
+            item["lon"], item["lat"] = location["geometry"]["coordinates"]
```

Comment:
For future reference, you can set `item["geometry"] = location["geometry"]` because we support GeoJSON geometries.